evm/README.md

```diff
@@ -316,8 +316,8 @@ from an instance of `NttManager` with rate-limiting enabled to `NttManagerNoRate
 
 #### Per-Chain-Transceivers
 
-The `NttManagerWithPerChainTransceivers` contract which inherits from `NttManagerNoRateLimiting` and allows configuring different transceivers and thresholds for each chain.
-You can configure a different set of send and receive transceivers for each chain. If you don't specifically enable any transceivers for a chain, then all transceivers will be used.
+The `NttManagerWithPerChainTransceivers` contract inherits from `NttManagerNoRateLimiting` and allows configuring different transceivers and thresholds for each chain.
+You can configure a different set of send and receive transceivers for each chain, as well as the receive threshold for the chain.
```

Comment:
Something that might be worth adding to the README.md: 

Even when only using per chain transceivers, one needs to register each transceiver using the `setTransceiver()` method inherited from `ManagerBase`. Note that this only allows registering up to 64 transceivers in total (to be used as send and/or receive transceivers).